client/blocks/sharing-preview-pane/index.jsx

```diff
@@ -111,7 +111,13 @@ class SharingPreviewPane extends PureComponent {
 
 		switch ( selectedService ) {
 			case 'facebook':
-				return <FacebookSharePreview { ...previewProps } />;
+				return (
+					<FacebookSharePreview
+						{ ...previewProps }
+						articleExcerpt={ post.excerpt }
```

Comment:
The Facebook preview handles truncation itself. We need to pass the raw excerpt and content to respect that.